app/views/projects/index.html.erb

```diff
@@ -9,9 +9,10 @@
 <div class="center">
 <h2>Projects</h2>
   <%= search_form_for @search do |f| %>
+      <%= f.label :badge_status %>
+      <%= f.collection_select(:badge_status_matches, Project::BADGE_STATUS_CHOICE, :id, :name) %>
```

Comment:
If you push your changes and add a link here, I'm happy to take a look and offer suggestions.